post-stack.yaml

```diff
@@ -169,6 +169,170 @@
     - include: tasks/devstack/check-devstack-cinder-services.yaml
       when: zuul_project == "openstack/cinder" or zuul_project == "openstack/os-brick"
 
+- hosts: win2016-compute
```

Comment:
I added a new play in post-stack.yaml because while using delegate_to, the **register** directive (eg. **register: neutron_service_check**) can not be used to the next task even though will be executed on the same host.